.github/workflows/cypress.yml

```diff
@@ -95,13 +95,13 @@ jobs:
       matrix:
         # Run multiple copies of the current job in parallel
         # Please increase the number or runners as your tests suite grows
-        containers: ['1', '2', '3']
+        containers: ['component', '1', '2', '3']
```

Comment:
@nickvergessen is this correct?

Reply:
You can drop it, else Cypress will complain.